temporal/t.remove/t.remove.py

```diff
@@ -100,19 +100,17 @@ def main():
         else:
             dataset_list = tuple(datasets.split(","))
 
-    # Read the dataset list from file
     if file:
-        fd = open(file)
-
         line = True
```

Comment:
I have no idea what the `line = True` is here for. Is the value ever used?

Reply:
Probably so that it is initialized inside the whole loop? But it shouldn't work as expected, as it is being set by readlines()